src/stores/expansion.ts

```diff
@@ -17,9 +17,13 @@ export const expansionSets = gqlSubscribable<ExpansionSet[]>(gql.SUB_EXPANSION_S
 
 export const creatingExpansionSequence: Writable<boolean> = writable(false);
 
-export const expansionRulesColumns: Writable<string> = writable('1fr 3px 2fr');
+export const expansionRulesColumns: Writable<string> = writable('2fr 3px 1fr');
```

Comment:
Is there a reason we flipped the sizing here?

Reply:
Reason for flipping sizes in expansion rules+sets, sequencing, scheduling, and constraints is to accommodate all the new metadata in the tables (went this direction after chatting with @lklyne). We could pursue hidden fields + the activity table column picker and I did go down this route for a bit but bailed due to the complexity of setting up a generic table with a column picker rendered _somewhere else_. Happy to chat more about it, maybe there's a simpler solution that doesn't involve lots of boilerplate.

Reply:
Oh and I'll note that while the sizings are flipped for the table heavy views, the editor views give more space to the editor.

Reply:
Okay thanks for the explanation!  I think this will be okay. My main worry is just not being able to see enough of the code. But they can always open the sequence to see more of the code, or just resize the panel.